packages/synapse-react-client/src/components/SynapseChat/SynapseChatInteraction.tsx

```diff
@@ -73,11 +80,31 @@ export const SynapseChatInteraction: React.FunctionComponent<
               borderRadius: '10px',
               padding: '10px',
               maxWidth: '100%',
+              whiteSpace: 'pre-wrap',
```

Comment:
any aversion to just sticking the output into a `MarkdownSynapse`? it seems to want to give us Markdown a lot (maybe just passing from a wiki)

Reply:
I thought about that too.  I think it would usually do the right thing, but was concerned about the cases when we process something that was not meant to be markdown syntax :/
I'll make it a MarkdownSynapse component :D

Reply:
Done!